fizzbuzz/typescript/Skeegan123/main.ts

```diff
@@ -9,10 +9,15 @@ function main(): void {
   }
 }
 
+export
 function fizzbuzz(n: number): string {
+  if (n % 1155 === 0) return "FizzBuzzPingPong";
+  if (n % 77 === 0) return "PingPong";
   if (n % 15 === 0) return "FizzBuzz";
   if (n % 3 === 0) return "Fizz";
   if (n % 5 === 0) return "Buzz";
+  if (n % 7 === 0) return "Ping";
+  if (n % 11 === 0) return "Pong";
```

Comment:
this could be done a bit differently.  build up a string and compare against the number only once.